jjb/egeria/egeria.yaml

```diff
@@ -11,7 +20,7 @@
       - 'master':
           branch: master
     jobs:
-      - '{project-name}-github-maven-jobs'
+      - egeria-maven-jobs
```

Comment:
Instead of adding a custom job group and then only using it in one place, fully spell out the jobs (and overrides) that the job group has here.

So, instead of
jobs:
  - egeria-maven-jobs

it would be
jobs:
  - github-maven-clm
  - github-maven-merge
  - github-mave-stage
  - github-maven-verify:
      mvn-goals: 'clean deploy -DskipFVT'